### PR TITLE
Updates overview.md

### PR DESCRIPTION
**Pull request by FrankieTF**
Update Microsoft.AspNetCore.DataProtection.AzureStorage and Microsoft.AspNetCore.DataProtection.AzureKeyVault sdk to track2 in secure-data_protection-configuration-overview.md.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=20329) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=20329) before we can accept your contribution.<br/><br/>:x: FrankieTF [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=20329)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=20329) it.</sub>

File: aspnetcore/security/data-protection/configuration/overview.md [hunk 1]
```diff
@@ -35,14 +35,20 @@ The following NuGet packages are required for the Data Protection extensions use
 
 ## ProtectKeysWithAzureKeyVault
 
+Log in to Azure using the CLI, you can type:
```
Comment (Rick-Anderson) [inline aspnetcore/security/data-protection/configuration/overview.md:38]:
```suggestion
Log in to Azure using the CLI, for example:
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@blowdart  please recommend a reviewer.

**Comment by jongio** [comment]
@pakrym - Can you please review?  Thanks

**Review by pakrym — APPROVED** [review]

**Comment by scottaddie** [comment]
Thank you for the PR, @FrankieTF!

**Comment by krish-gh** [comment]
#20112

**Pull request merged**